src/nni_manager/training_service/pai/paiK8S/paiK8SData.ts

```diff
@@ -31,6 +31,6 @@ fi`;
 
 export const PAI_K8S_TRIAL_COMMAND_FORMAT: string =
 `export NNI_PLATFORM=pai NNI_SYS_DIR={0} NNI_OUTPUT_DIR={1} NNI_TRIAL_JOB_ID={2} NNI_EXP_ID={3} NNI_TRIAL_SEQ_ID={4} MULTI_PHASE={5} \
-&& NNI_CODE_DIR={6} && cp -r $NNI_CODE_DIR/. $NNI_SYS_DIR && cd $NNI_SYS_DIR && sh install_nni.sh \
-&& python3 -m nni_trial_tool.trial_keeper --trial_command '{7}' --nnimanager_ip '{8}' --nnimanager_port '{9}' \
+&& NNI_CODE_DIR={6} && mkdir -p $NNI_SYS_DIR/code && cp -r $NNI_CODE_DIR/. $NNI_SYS_DIR/code && sh $NNI_SYS_DIR/install_nni.sh \
```

Comment:
why need extra code folder?

Reply:
If to avoid conflict, we can rename `run.sh` to `nni_run.sh` to avoid it.

Reply:
As discussed with @QuanluZhang , we use a `code` folder to store users' code files. In this way, NNI will not destroy users' folder structure.

Reply:
How about parameter file? It needs to manual run and verify it at least on remote, and pai.

Reply:
never mind, I had a check. It looks good, if test cases doesn't timeout.